content/pt-br/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -18,122 +15,93 @@ princípios do Kubernetes, notavelmente o [ciclo de controle](/docs/concepts/#ku
-que pode atuar como um [cliente da API do Kubernetes](/docs/reference/using-api/client-libraries/).
+A seguir estão algumas bibliotecas e ferramentas que você pode usar para escrever seu próprio operador nativo de nuvem.
 
+> **_NOTE:_**  Esta seção contém links para projetos de terceiros que fornecem funcionalidades requeridas pelo Kubernetes. Os autores do projeto Kubernetes não são responsáveis por esses projetos, que estão listados em ordem alfabética. Para adicionar um projeto a esta lista, leia o [guia de conteúdo](/docs/contribute/style/content-guide/#third-party-content) antes de enviar uma alteração. [Mais informações](/docs/concepts/extend-kubernetes/operator/#third-party-content-disclaimer).
 
+{{% thirdparty-content %}}
 
-## {{% heading "whatsnext" %}}
+* [Charmed Operator Framework](https://juju.is/)
+* [Java Operator SDK](https://github.com/java-operator-sdk/java-operator-sdk)
+* [Kopf](https://github.com/nolar/kopf) (Kubernetes Operator Pythonic Framework)
+* [kube-rs](https://kube.rs/) (Rust)
+* [kubebuilder](https://book.kubebuilder.io/)
+* [KubeOps](https://buehler.github.io/dotnet-operator-sdk/) (.NET operator SDK)
+* [KUDO](https://kudo.dev/) (Kubernetes Universal Declarative Operator)
+* [Mast](https://docs.ansi.services/mast/user_guide/operator/)
+* [Metacontroller](https://metacontroller.github.io/metacontroller/intro.html) juntamente com *WebHooks* que você mesmo implementa
+* [Operator Framework](https://operatorframework.io)
+* [shell-operator](https://github.com/flant/shell-operator)
 
+## O que vem a seguir
 
-* Aprenda mais sobre [Recursos Personalizados](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-* Encontre operadores prontos em [OperatorHub.io](https://operatorhub.io/) para o seu caso de uso
-* Use ferramentes existentes para escrever os seus Operadores:
-  * usando [KUDO](https://kudo.dev/) (Kubernetes Universal Declarative Operator)
-  * usando [kubebuilder](https://book.kubebuilder.io/)
-  * usando [Mast](https://docs.ansi.services/mast/user_guide/operator/)
-  * usando [Metacontroller](https://metacontroller.github.io/metacontroller/intro.html) juntamente com WebHooks que
-    implementa você mesmo
-  * usando o [Operator Framework](https://operatorframework.io/)
-* [Publique](https://operatorhub.io/) o seu operador para que outras pessoas o possam usar
-* Leia o [artigo original da CoreOS](https://coreos.com/blog/introducing-operators.html) que introduz o padrão Operador
-* Leia um [artigo](https://cloud.google.com/blog/products/containers-kubernetes/best-practices-for-building-kubernetes-operators-and-stateful-apps) da Google Cloud sobre as melhores práticas para contruir Operadores
 
+* Leia o {{< glossary_tooltip text="CNCF" term_id="cncf" >}}
+  [Operator White Paper](https://github.com/cncf/tag-app-delivery/blob/eece8f7307f2970f46f100f51932db106db46968/operator-wg/whitepaper/Operator-WhitePaper_v1-0.md).
```

Comment:
How would you feel about changing this to "Leia o whitepaper sobre operadores da CNCF" (or another wording that delivers similar meaning in Portuguese)?